apps/language_server/test/providers/execute_command/manipulate_pipes/ast_test.exs

```diff
@@ -0,0 +1,159 @@
+defmodule ElixirLS.LanguageServer.Providers.ExecuteCommand.ManipulatePipes.ASTTest do
```

Comment:
Most of these test I imported from my original repo, so if any of these tests feel redundant, feel free to make suggestions for me to remove them!